sql/size_utils.sql

```diff
@@ -682,3 +635,126 @@ BEGIN
 END;
 $BODY$;
 
+-------------Get index size for hypertables -------
+-- hypertable_index_name      - index on hyper table
+CREATE OR REPLACE FUNCTION _timescaledb_internal.indexes_local_size(
+    index_name_in              NAME
+)
+RETURNS TABLE ( hypertable_id INTEGER,
```

Comment:
@mkindahl This function is internal, not for public use.

Reply:
name is definitely not the right datatype cause it has 64 char limit but since both table name and schema can be 64 chars its too short, regclass would be more appropriate or text if we dont want oid but i would prefer regclass here

Reply:
seems to be missing schema, i think regclass would be more appropriate here

Reply:
added schema name. Needs to be used across data nodes, so cannot use regclass here.

Reply:
Not sure I follow: why would regclass not work? The parameter to `drop_chunks` is regclass and it works.

Reply:
we can move to regclass when the catalogs use regclass. The regclass here needs to be translated into schema+name to effectively query our catalogs. We do not want to do the mapping twice: once in the external function and then again in the internal function.